examples/tti_example.py

```diff
@@ -103,22 +103,25 @@ def damp_boundary(damp, h, nbpml):
             damp[:, :, -(i + 1)] += val
 
 nrec, _ = data.traces.shape
-m = DenseData("m", model.vp.shape, dtype)
+m = DenseData(name="m", shape=model.vp.shape, dtype=dtype)
```

Comment:
Uhh advanced inflection, I am intrigued. :smile: Yes, probably, but I don't quite think this is necessary right now.